eg/quickblox-bot/index.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+// Example QuickBlox bot for RiveScript-JS.
+//
+// To run this bot, edit 'CONFIG' variable below to fill in your QuickBlox settings for the bot.
+// 
+// Run this demo: `node index.js`
+
+const QB = require('quickblox');
+
+// This would just be require("rivescript") if not for running this
+// example from within the RiveScript project.
+const RiveScript = require("../../lib/rivescript");
+
+// In order to start develop you first Chat Bot you have to register new QuickBlox account and create your first application.
+// Go to https://quickblox.com/signup and get a QuickBlox account, then go to https://admin.quickblox.com/apps/new
+// and create your first application. Then put here Application ID, Authorization key, Authorization secret.
+//
+// also
+//
+// Go to Users module in QuickBlox dashboard
+// (e.g. <your_app_id>/service/users https://admin.quickblox.com/apps/<your_app_id>/service/users) and create
+// new user for you chat bot. Then put here user's ID, password, login and full name.
+//
+const CONFIG = {
+    "appId": "...",
+    "authKey": "...",
+    "authSecret": "...",
+    "user": {
+        "id": "...",
+        "login": "...",
+        "password": "...",
+        "fullname": "..."
+    }
+};
+
+// Init RiveScript logic
+var riveScriptGenerator = new RiveScript();
+
+function loadingDone(batch_num) {
+    console.log(`[RiveScript] Batch #${batch_num} has finished loading!`);
+    riveScriptGenerator.sortReplies();
+}
+
+function loadingError(batch_num, error) {
+    console.log(`[RiveScript] Load the batch #${batch_num} is failed`, JSON.stringify(error));
+}
+
+riveScriptGenerator.loadFile('replies.rive', loadingDone, loadingError);
```

Comment:
Can you change this to do `.loadDirectory('../brain')` instead? The contents of your `replies.rive` is basically all the files in `/eg/brain` concatenated together into one, so have it just use the source files in that directory instead.

(And remove `replies.rive` from this pull request)